entity-service/src/main/kotlin/com/egm/stellio/entity/web/EntityHandler.kt

```diff
@@ -94,6 +97,10 @@ class EntityHandler(
         val contextLink = getContextFromLinkHeaderOrDefault(httpHeaders)
         val mediaType = getApplicableMediaType(httpHeaders)
 
+        if (limit <= 0 || page <= 0)
+            return ResponseEntity.status(HttpStatus.BAD_REQUEST).contentType(MediaType.APPLICATION_JSON)
+                .body(BadRequestDataResponse("Page number and Limit must be greater than zero"))
```

Comment:
greater or strictly greater? what if I set limit to 0? page starts at 0 or at 1?

Reply:
Limit and page, both, must be strictly greater than 0, no ?

Reply:
yes, strictly greater.